Makefile

```diff
@@ -78,7 +78,7 @@ test-style: check-docker
 
 # This should only be executed within the containerized development environment.
 style-check:
-	lint
+	lint --deadline=120m
```

Comment:
What is the `--deadline=120m` supposed to do?

If you look at https://github.com/deis/docker-go-dev/blob/v1.10.0/rootfs/usr/local/bin/lint, it looks like arguments to the shell script are not passed to `gometalinter` at all, so this doesn't make any sense.

Reply:
If we want to increase the deadline, we would need to override the `/usr/local/bin/lint` shell script with a patched version, either by hardcoding a higher value or passing `"$@"` to the `gometaliner` calls.

However since the default is 20 seconds, increasing to two *hours* is probably a bad idead for CI, 1m or 2m would probably be saner.

Reply:
Btw. the following works, if you don't want to copy `/usr/local/bin/lint`:

```diff
--- a/Makefile
+++ b/Makefile
@@ -78,7 +78,7 @@ test-style: check-docker

 # This should only be executed within the containerized development environment.
 style-check:
-       lint
+       sed -e 's/--deadline=20s/--deadline=2m/g' /usr/local/bin/lint | bash
        shellcheck $(SHELL_SCRIPTS)

 test-unit:
```

It's a bit hacky though… ;-)

Reply:
Thank you for taking a look at this. This was happening on my systems as I'm building Docker containers with less than 4GB of memory and linter was failing as the container was running out of memory. As this seems like only a local problem to me, I am going to drop this commit and figure out another way to speed up the Go linter.